src/TMainConsole.cpp

```diff
@@ -54,6 +54,7 @@ TMainConsole::TMainConsole(Host* pH, QWidget* parent)
 , mSpellDic()
 , mpHunspell_system(nullptr)
 , mpHunspell_shared(nullptr)
+, mpHunspellCodec_system(nullptr)
```

Comment:
:information_source: This needs to be ***below*** the next line down so that they are initialised in the correct order (Qt Creator may light it up as with a warning depending on your Qt Creator code model/checking settings).

Reply:
Not in this PR, but one day we should change it all to the modern brace initialisation syntax so we don't have to play these games where we get the order of this right.